ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -1,12 +1,17 @@
 (ns cmr.ingest.services.bulk-update-service
   (:require
+   [camel-snake-kebab.core :as csk]
    [cheshire.core :as json]
    [clojure.java.io :as io]
+   [clojure.string :as str]
```

Comment:
I think we are changing our naming convention to use `string` as the alias for `clojure.string` as suggested by @oubiwann.